spark/src/main/scala/geotrellis/spark/summary/polygonal/RDDPolygonalSummary.scala

```diff
@@ -79,7 +79,7 @@ object RDDPolygonalSummary {
         val extent: Extent = layout.mapTransform.keyToExtent(spatialKey)
         val raster: Raster[T] = Raster(tile, extent)
         val result: PolygonalSummaryResult[R] =
-          raster.polygonalSummary(feature.geom, visitor.getClass.newInstance, options)
+          raster.polygonalSummary(feature.geom, visitor.getClass.getDeclaredConstructor().newInstance(), options)
```

Comment:
@echeipesh what is the difference?

Reply:
https://docs.oracle.com/javase/9/docs/api/java/lang/Class.html#newInstance--
> Deprecated. This method propagates any exception thrown by the nullary constructor, including a checked exception. Use of this method effectively bypasses the compile-time exception checking that would otherwise be performed by the compiler. The Constructor.newInstance method avoids this problem by wrapping any exception thrown by the constructor in a (checked) InvocationTargetException.